src/components/IOList.tsx

```diff
@@ -60,6 +60,7 @@ const IOList = ({ currentAddress, isOut, outputs, inputs, timestamp, linkToExplo
 
     const getAddressComponent = (addressHash: AddressHash) => {
       const address = getAddress(addressHash)
+      if (!address) return null
```

Comment:
That means that the following condition will never evaluate to display the component

```ts
<AddressEllipsed addressHash={addressHash} />
```

...

Reply:
If the hash is not one of the internal addresses, your change returns `null` instead of the ellipsed hash.